sssd_test_framework/roles/ipa.py

```diff
@@ -1597,3 +1601,88 @@ def __get_info(self, info: str | NFSExport | IPAAutomountMap | None) -> str | No
             return info.name
 
         return info
+
+
+class IPAIdView(IPAObject):
+    """
+    IPA id view management.
```

Comment:
nitpick, keep the spelling consistent, below you have 

```IPA ID view```, here it's ```IPA id view```

Reply:
ack